python/setup.py

```diff
@@ -48,5 +48,6 @@
         path.join(UTILITIES_PATH, "submit_recovery_share.sh"),
         path.join(UTILITIES_PATH, "verify_quote.sh"),
     ],
+    entry_points={"console_scripts": ["ccf_cose_sign1 = ccf.cose:sign_cli"]},
```

Comment:
Do we want to have test for `ccf_cose_sign1`? I think it would be helpful as example usage not only preventing degradation.

Reply:
Yes we do!

Reply:
I'm updating the CurlClient to have a path that uses ccf_cose_sign1, that'll make it easier to generate the doc and test the tool end to end.

Reply:
@takuro-sato done now, this is tested by the sandbox/test_install* scripts, rather than the main tests themselves. We use to run some of them with curl client too, but they're very slow and not that interesting.

Reply:
I feel like I am missing something, but I can't find any file sandbox/test_install* that has been changed. Did you forget to upload the change?

Reply:
They have not, but what they do is they use the sandbox, which now sets CURL_CLIENT=1 and CURL_CLIENT_USE_COSE=1, and the CurlClient has been updated to use ccf_cose_sign1 when those are set. Another way to see that in action is to run the sandbox.